Author: Diane Trout <diane@ghic.org>
Description: Use local files for building documentation instead of
 downloading them from github.
Forwarded: no

--- dask.orig/docs/source/configuration.rst
+++ dask/docs/source/configuration.rst
@@ -465,8 +465,8 @@
 
 .. dask-config-block::
     :location: dask
-    :config: https://raw.githubusercontent.com/dask/dask/main/dask/dask.yaml
-    :schema: https://raw.githubusercontent.com/dask/dask/main/dask/dask-schema.yaml
+    :config: file:///../debian/dask-config-blocks/dask.yaml
+    :schema: file:///../debian/dask-config-blocks/dask-schema.yaml
 
 
 Distributed Client
@@ -474,16 +474,16 @@
 
 .. dask-config-block::
     :location: distributed.client
-    :config: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed.yaml
-    :schema: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed-schema.yaml
+    :config: file:///../debian/dask-config-blocks/distributed.yaml
+    :schema: file:///../debian/dask-config-blocks/distributed-schema.yaml
 
 Distributed Comm
 ~~~~~~~~~~~~~~~~
 
 .. dask-config-block::
     :location: distributed.comm
-    :config: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed.yaml
-    :schema: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed-schema.yaml
+    :config: file:///../debian/dask-config-blocks/distributed.yaml
+    :schema: file:///../debian/dask-config-blocks/distributed-schema.yaml
 
 
 Distributed Dashboard
@@ -491,8 +491,8 @@
 
 .. dask-config-block::
     :location: distributed.dashboard
-    :config: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed.yaml
-    :schema: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed-schema.yaml
+    :config: file:///../debian/dask-config-blocks/distributed.yaml
+    :schema: file:///../debian/dask-config-blocks/distributed-schema.yaml
 
 
 Distributed Deploy
@@ -500,8 +500,8 @@
 
 .. dask-config-block::
     :location: distributed.deploy
-    :config: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed.yaml
-    :schema: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed-schema.yaml
+    :config: file:///../debian/dask-config-blocks/distributed.yaml
+    :schema: file:///../debian/dask-config-blocks/distributed-schema.yaml
 
 
 Distributed Scheduler
@@ -509,8 +509,8 @@
 
 .. dask-config-block::
     :location: distributed.scheduler
-    :config: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed.yaml
-    :schema: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed-schema.yaml
+    :config: file:///../debian/dask-config-blocks/distributed.yaml
+    :schema: file:///../debian/dask-config-blocks/distributed-schema.yaml
 
 
 Distributed Worker
@@ -518,8 +518,8 @@
 
 .. dask-config-block::
     :location: distributed.worker
-    :config: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed.yaml
-    :schema: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed-schema.yaml
+    :config: file:///../debian/dask-config-blocks/distributed.yaml
+    :schema: file:///../debian/dask-config-blocks/distributed-schema.yaml
 
 
 Distributed Nanny
@@ -527,8 +527,8 @@
 
 .. dask-config-block::
     :location: distributed.nanny
-    :config: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed.yaml
-    :schema: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed-schema.yaml
+    :config: file:///../debian/dask-config-blocks/distributed.yaml
+    :schema: file:///../debian/dask-config-blocks/distributed-schema.yaml
 
 
 Distributed Admin
@@ -536,8 +536,8 @@
 
 .. dask-config-block::
     :location: distributed.admin
-    :config: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed.yaml
-    :schema: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed-schema.yaml
+    :config: file:///../debian/dask-config-blocks/distributed.yaml
+    :schema: file:///../debian/dask-config-blocks/distributed-schema.yaml
 
 
 Distributed RMM
@@ -545,5 +545,5 @@
 
 .. dask-config-block::
     :location: distributed.rmm
-    :config: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed.yaml
-    :schema: https://raw.githubusercontent.com/dask/distributed/main/distributed/distributed-schema.yaml
+    :config: file:///../debian/dask-config-blocks/distributed.yaml
+    :schema: file:///../debian/dask-config-blocks/distributed-schema.yaml
